Content.Shared/Overlays/Switchable/ThermalVisionComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Shared.Actions;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Overlays.Switchable;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ThermalVisionComponent : SwitchableOverlayComponent
+{
+    public override string? ToggleAction { get; set; } = "ToggleThermalVision";
+
+    public override Color Color { get; set; } = Color.FromHex("#F84742");
```

Comment:
```suggestion

    [DataField]
    public override Color Color { get; set; } = Color.FromHex("#F84742");
```
Same here, I wanna do custom color thermographics. :)

Reply:
ThermalVisionComponent inherits the Color variable from BaseOverlayComponent; there's no need to use DataField. It works fine without it, as can be verified by looking at the prototypes.